e2e/pages/adf/cardViewComponentPage.ts

```diff
@@ -218,4 +218,13 @@ export class CardViewComponentPage {
         }
     }
 
+    async isErrorNotDisplayed(): Promise<boolean> {
+        const errorElement: ElementFinder = element(by.css('mat-error[data-automation-id="card-textitem-error-int"]'));
+        return  BrowserVisibility.waitUntilElementIsNotVisible(errorElement);
```

Comment:
this will not return always a boolean you need somethign like that:
try {
      await BrowserVisibility.waitUntilElementIsVisible(this.header);
      return true;
    } catch {
      return false;
    }

Reply:
I've added the try-catch block, please take a look